### DISCUSSION TITLE
new releases : force to upgrade Selenium webdriver to >4 ?

### DISCUSSION THREAD

**Discussion by juprog** [TroyWalshProf/SeleniumAxeDotnet#225] [General]
Hi, 
I am not sure , but it looks like when I try to upgrade from nuget, to a latest version of SeleniumAxeDotNet, it forces a dependency on SeleniumWebdriver 4...   
does that mean we are forced to upgrade to that if we want to keep SeleniumAxeDotNet updated?
 and it no longer supports 3.141 ?

**Comment by TroyWalshProf** [comment]
Short answer is yes, Selenium 4 is required.
*One of the breaking changes required us to either support 3 or 4.

For more info see: https://github.com/TroyWalshProf/SeleniumAxeDotnet/discussions/208

**Reply by TroyWalshProf** [reply]
If you really want to use an older version of Selenium you can do something like this:
https://duanenewman.net/blog/post/forcing-a-specific-target-platform-with-packagereference/
Note* The Html report create functionality will not work if you use this hack

**Comment by juprog** [comment]
ok. Thank you for confirming it was a breaking change, and that you had to choose to support one or the other, and the quick reply.
